baseline_model.py

```diff
@@ -0,0 +1,41 @@
+import Read_data_and_Write_results as rw
+from word_frequency_model import WordFrequencyModel as wfm
+import argparse
+import os
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-tr', '--train', help="path to semi_train.txt dataset")
+    parser.add_argument('-ts', '--test', help="path to test.txt dataset")
+    parser.add_argument('-o', help="path to submition output input/res",default="input")
+    args = parser.parse_args()
+    if args.train is None or args.test is None:
+        parser.print_usage()
+        exit()
+    return args
+
+def mkdir(path):
+    if not os.path.exists(path):
+        os.mkdir(path)
+
+
+if __name__ == "__main__":
+    args = get_args()
+    path_to_train = os.path.normpath(args.train) #path to semi_train.txt
+    path_to_output = os.path.normpath(args.o)  #path to  input/res submition dir
+    path_to_test = os.path.normpath(args.test) #path to test.txt
+    
+    mkdir(path_to_output)
+    path_to_output = os.path.join(path_to_output , "res")
+    mkdir(path_to_output)
+
+    _, word_lsts, _, bio_lsts, _, _  = rw.read_data(path_to_train)
+
+    print("training the model ... ")    
+    wfm_model = wfm(word_lsts, bio_lsts)
+    wfm_model.fit()
```

Comment:
Sorry to bother you again. The inputs and labels should be the parameters of `fit` not the class constructor.

Reply:
We did it. Is that you meant?